schemas/openapi.yml

```diff
@@ -0,0 +1,19 @@
+openapi: 3.1.0
+info:
+  title: Federated API Discovery API
+  version: v0
+  description: This produces metadata about APIs published by public sector organisations
+    in the UK
+paths:
+  "/apis":
+    get:
+      description: Returns metadata about APIs published by public sector organisations
+        in the UK
+      operationId: retrieveAll
+      responses:
+        '200':
+          description: OK
```

Comment:
This should be a descriptive reason of what's occurring, rather than describing the HTTP status